ui/docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+version: "3"
+services:
+  ui:
+    image: node
+    working_dir: /w
+    volumes:
+      - .:/w:cached
+    command: npm start -- --host=0.0.0.0 ${ENV_FLAG}
+    ports:
+      - 4200:4200
```

Comment:
I wonder if we should define a repo-wide docker-compose that can launch server + client together.

Benefits: can launch a full stack at once; can mount the full source into all containers so, e.g. ui container can see swagger file.

Challenges: we lose strict separation/isolation between components

On the whole, I think a repo-wide compose file might be convenient.